src/views/Characters/Characters.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from 'react';
+
+import { fetchCharacters } from '../../services/characters';
+import SearchRace from '../../components/SearchRace';
+
+
+export default function Characters() {
+  const [stars, setStars] = useState([]);
+  const [race, setRace] = useState('All');
+  useEffect(()=> {
+    const everyStarFunct = async () => {
+      const everyStar = await fetchCharacters(race);
+      setStars(everyStar);
+    };
```

Comment:
It's always a good idea to have a try/catch block when calling a third party API, in case the API site is down.